### DISCUSSION TITLE
Dataset Application

### DISCUSSION THREAD

**Discussion by xmh1011** [xmh1011/AT-DGNN#14] [Q&A] [answered]
You can leave your email here. Waiting for the paper to be accepted, we will release the dataset and let you know ASAP. Thank you for your attention. I hope we can keep in touch and communicate in time if there are any questions.

**Comment by LoongKing6** [comment]
大哥，我需要数据集，邮箱：3303774971@qq.com
希望你们这篇论文能中。
谢谢！

**Reply by xmh1011** [reply]
https://drive.google.com/drive/folders/1Tabw5sjpFiwy88yP-C-LnunNFrrre9AR?usp=drive_link

**Comment by xmh1011** [comment, marked as answer]
dataset: https://drive.google.com/drive/folders/1Tabw5sjpFiwy88yP-C-LnunNFrrre9AR?usp=drive_link

**Comment by LoongKing6** [comment]
谢谢，恭喜你们论文成功被接受，哈哈哈，我目前看你们论文已经几个星期了，你们论文真的很详细，我的导师说，我研一上能把你们论文和代码完全看懂就够了。
&nbsp;


fanjinke7
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年11月20日(星期三) 下午2:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [xmh1011/AT-DGNN] Dataset Application (Discussion #14)















关闭#14已解决。

-
直接回复这封邮件，在GitHub上查看，或取消订阅.
***@***.*** &gt;
